providers/os/resources/fstab.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+package resources
+
+import (
+	"bufio"
+	"errors"
+	"io"
+	"strconv"
+	"strings"
+
+	"go.mondoo.com/cnquery/v11/llx"
+	"go.mondoo.com/cnquery/v11/providers/os/connection/shared"
+)
+
+func (f *mqlFstab) entries() ([]any, error) {
+	conn := f.MqlRuntime.Connection.(shared.Connection)
+	fstabFile, err := conn.FileSystem().Open("/etc/fstab")
```

Comment:
i thihnk we should check if `conn.FileSystem()` returns an actual value. maybe some connections return `nil` here